### PR TITLE
Fix zeroctl copy : vdisk has tlog support if it has non empty tlogServerClusterId

### PR DESCRIPTION
**Pull request by iwanbk**
Basically do the same thing as #479 for alpha-8, but `copy tlog` code has been changed a lot in master, so need different PR

### CODE REVIEW

File: tlog/copy/copy.go [hunk 1]
```diff
@@ -47,15 +60,22 @@ func Copy(ctx context.Context, confSource config.Source, conf Config) error {
...
 	}
 
+	sourceVdiskNBDConfig, err := config.ReadVdiskNBDConfig(confSource, conf.SourceVdiskID)
+	if err != nil {
+		return fmt.Errorf(
+			"couldn't read source vdisk %s's storage config: %v", conf.SourceVdiskID, err)
+	}
 	// if source vdisk has no tlog, generate it
-	if !sourceStaticConf.Type.TlogSupport() {
+	if !sourceStaticConf.Type.TlogSupport() || sourceVdiskNBDConfig.TlogServerClusterID == "" {
```
Comment (zaibon) [inline tlog/copy/copy.go]: shouldn't this be the opposite? If not tlog configured, we don't do anything?

Comment (zaibon) [reply tlog/copy/copy.go]: But this should only happens if the source vdisk doesn't have tlog data, but the target vdisk has a tlog configured right ?

Comment (iwanbk) [reply tlog/copy/copy.go]: Yeah, check for target vdisk also in this PR https://github.com/zero-os/0-Disk/pull/480/files#diff-0e86d341a795fa64360e611d587a8170R49

Comment (zaibon) [reply tlog/copy/copy.go]: ok LGTM

**Comment by GlenDC** [comment]
This PR is no longer required due to the merging of the original PR (#479).

**Pull request closed without merging**